app/Http/Controllers/Auth/AuthController.php

```diff
@@ -179,8 +185,55 @@ public function postOauth(Request $request)
         return redirect($this->redirectPath());
     }
 
+    /**
+     * Show form to collect new username.
+     * 
+     * @return Response
+     */
     public function getOauth()
     {
         return view('/errors/oauthname');
     }
+
+    /**
+     * Send a mail to register a user.
+     * 
+     * @param  Request $request
+     * @return Response
+     */
+    public function sendMail(Request $request)
+    {
+        $validator = $this->validator($request->all());
+
+        if ($validator->fails()) {
+            $this->throwValidationException(
+                $request, $validator
```

Comment:
Which multi-line function call?

If it is the "$this->throwValidationException" function call.

That's exactly how the Laravel framework implements that line in other
methods where it is used.

On Mon, Nov 23, 2015 at 11:25 AM, Nitpick <notifications@github.com> wrote:

> In app/Http/Controllers/Auth/AuthController.php
> <https://github.com/andela/pibbble/pull/141#discussion_r45588078>:
>
> >      public function getOauth()
> >      {
> >          return view('/errors/oauthname');
> >      }
> > +
> > +    /**
> > +     * Send a mail to register a user.
> > +     *
> > +     * @param  Request $request
> > +     * @return Response
> > +     */
> > +    public function sendMail(Request $request)
> > +    {
> > +        $validator = $this->validator($request->all());
> > +
> > +        if ($validator->fails()) {
> > +            $this->throwValidationException(
> > +                $request, $validator
>
> Only one argument is allowed per line in a multi-line function call
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/andela/pibbble/pull/141/files#r45588078>.
>



-- 
*Opeyemi Ogunjimi | Software Developer | Andela*
(c) +234-703-936-2475 | Andela.com <http://andela.com/>